likes.rb

```diff
@@ -0,0 +1,18 @@
+def likes(names)
+
+  if names.empty?
```

Comment:
This is better as an `if else elsif else` since the branches are all mutually exclusive. Meaning, the names array count cannot be _both_ 1 and 2.

```
if names.count == 1
  # ...
elsif names.count == 2
  # ...
else
  # ...
end
```